gogs_client/entities.py

```diff
@@ -240,3 +240,46 @@ def pull(self):
             :rtype: bool
             """
             return self._pull
+
+
+class TokenInfo(object):
```

Comment:
Does this need to be a separate class? Can we just add a `name` property to the existing `Token` class?